pkg/models/governance_request_feedback.go

```diff
@@ -0,0 +1,32 @@
+package models
+
+import "github.com/google/uuid"
+
+// GovernanceRequestFeedbackSourceAction is an eumeration of the possible actions that can provide feedback on a governance request
+type GovernanceRequestFeedbackSourceAction string
+
+// These are the possible forms that can provide feedback
+const (
+	GovernanceRequestFeedbackSourceActionRequestEdits      GovernanceRequestFeedbackSourceAction = "REQUEST_EDITS"
+	GovernanceRequestFeedbackSourceActionProgressToNewStep GovernanceRequestFeedbackSourceAction = "PROGRESS_TO_NEW_STEP"
+)
+
+// GovernanceRequestFeedbackTargetForm is an enumeration of the possible forms on a governance intake request that can receive feedback
+type GovernanceRequestFeedbackTargetForm string
+
+// These are the possible forms that can have governance request feedback
+const (
+	GovernanceRequestFeedbackTargetNoTargetProvided  GovernanceRequestFeedbackTargetForm = "NO_TARGET_PROVIDED"
+	GovernanceRequestFeedbackTargetIntakeRequest     GovernanceRequestFeedbackTargetForm = "INTAKE_REQUEST"
+	GovernanceRequestFeedbackTargetDraftBusinessCase GovernanceRequestFeedbackTargetForm = "DRAFT_BUSINESS_CASE"
+	GovernanceRequestFeedbackTargetFinalBusinessCase GovernanceRequestFeedbackTargetForm = "FINAL_BUSINESS_CASE"
+)
+
+// GovernanceRequestFeedback represents feedback given to the requester on a governance request
+type GovernanceRequestFeedback struct {
+	BaseStruct
+	IntakeID     uuid.UUID `db:"intake_id"`
+	Feedback     string
```

Comment:
I noticed that these fields don't all have struct tags (and they only have them for db and not json as well). Am I guessing correctly that sqlx a GQL are able to automatically convert the name if the name is one word?

 In `MINT` every struct has both tags provided on every field, but it looks like this is consistent with other locations in `EASi`

Reply:
Yeah, they both have some defaults, which we generally rely on in EASi:
- sqlx just calls `strings.ToLower()` on the field name. See http://jmoiron.github.io/sqlx/, "Controlling Name Mapping"
- I'm not sure if gqlgen configures its own defaults or if they're set up in the `encoding/json` package from the Go stdlib, but they're definitely present.

We're not entirely consistent about it, though, and I wouldn't be against including both tags as regular practice, especially if that's what MINT is doing as well.

Reply:
Awesome, thanks Dylan! I like including both struct tags myself. If you like it too, I think it could be a nice to adopt as a regular practice for EASi (I'll leave it up to you if you want to implement in this PR)

Reply:
Added `json` and `db` struct tags for all fields in https://github.com/CMSgov/easi-app/pull/2067/commits/50cae8fdec74bc25f960bd40507e015def41943c, since I was making edits anyways.